insights/parsers/swift_conf.py

```diff
@@ -0,0 +1,42 @@
+"""
+SwiftProxyServerConf - file ``/etc/swift/proxy-server.conf``
```

Comment:
AFAIK, in the `/etc/swift/` folder, there are also some configuration files like `swift.conf` and `object-server.conf`, and I think in the future some of them will be included in our framework. So, would you please modify the document more compatible?